frontend/src/pages/devices/List.tsx

```diff
@@ -153,6 +155,14 @@ export const List = () => {
         return <>{ level }%</>;
     };
 
+    const renderOnline = (entity: DeviceT) => {
+        if (entity.rawDevice.status === "ONLINE") {
+            return <Badge>Online</Badge>;
+        }else{
```

Comment:
Because of the `return` in your `if`, you can kill the `else` block, i.e.

```js
if (entity..... "ONLINE") {
  return whatever;
}
return theOtherThing;
```

Reply:
That makes sense now